release/_publishing.sh

```diff
@@ -317,18 +317,34 @@ function upload_release {
 }
 
 function upload_to_docker_hub {
-	_update_bundles_yml
+	prepare_branch_to_commit_from_master "${_PROJECTS_DIR}/liferay-docker" "liferay-docker" "update-bundles-yml-branch"
```

Comment:
@kiwm, please try to use a branch name with some unique identifier to prevent issues.